server/channels/app/helper_test.go

```diff
@@ -446,6 +452,10 @@ func (th *TestHelper) CreateGroupChannel(c request.CTX, user1 *model.User, user2
 }
 
 func (th *TestHelper) CreatePost(channel *model.Channel) *model.Post {
+	return th.CreatePostWithModifier(channel, noOpPostModifier)
+}
+
+func (th *TestHelper) CreatePostWithModifier(channel *model.Channel, postModifier postModifierFunc) *model.Post {
```

Comment:
I really like this approach to letting people customize since we can't just write `...override` like we can in the web app's test helper. I went to check to see if we did the same elsewhere in the server code, and I found that `TestHelper.Setup` and `TestHelper.CreateChannel` supports something similar.

Could you change this so that `CreatePost` can take any number of functions at the end and rename `postModifierFunc` to `PostOption` to be more similar to those?

Reply:
Done.